app/Jobs/PushServerUpdateJob.php

```
@@ -503,20 +504,28 @@ private function updateDatabaseStatus(string $databaseUuid, string $containerSta
+            return;
         }
+
+        $notFoundDatabaseUuids->each(function ($databaseUuid) {
+            $database = $this->databases->where('uuid', $databaseUuid)->first();
+            if ($database) {
+                if (! str($database->status)->startsWith('exited')) {
+                    $database->status = 'exited';
+                    $database->save();
+                }
+                if ($database->is_public) {
+                    StopDatabaseProxy::dispatch($database);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Actions\Database\StopDatabaseProxy' in method 'updateNotFoundDatabaseStatus'. (undefined)

(StaticAccess)